libraries/AP_Logger/AP_Logger_Backend.cpp

```diff
@@ -429,6 +429,13 @@ bool AP_Logger_Backend::ShouldLog(bool is_critical)
         return false;
     }
 
+    if (_front._last_mavlink_log_transfer_message_handled_ms != 0 &&
+        !_front.vehicle_is_armed() &&
+        AP_HAL::millis() - _front._last_mavlink_log_transfer_message_handled_ms < 10000) {
+        // user is transfering files via mavlink
+        return false;
+    }
+
```

Comment:
need to reset to avoid 10s gap in log in future

Reply:
Fixed.